internal/store/utils.go

```diff
@@ -125,6 +128,16 @@ func sanitizeLabelName(s string) string {
 	return invalidLabelCharRE.ReplaceAllString(s, "_")
 }
 
+func lintLabelName(s string) string {
+	return toSnakeCase(s)
+}
+
+func toSnakeCase(s string) string {
+	snake := matchFirstCap.ReplaceAllString(s, "${1}_${2}")
```

Comment:
I am trying to comprehend the use of this regex over here. I had commented out this line and the test cases still pass. Can you add another test case which would depend on the "match the first capital letter" regex? This would help me understand the use of this regex

Reply:
This regex catches strings starting with a word in full caps such as APPVersion, TLSVersion... It might be a bit too overengineered for our use case as such labels are very unlikely to be used.
I will update the PR and remove this regex. Thanks for finding this :slightly_smiling_face: .

Reply:
@tariq1890 I updated the PR, PTAL